From bf081add390e75a034ad6cdce3f284343e78fb26 Mon Sep 17 00:00:00 2001
From: Fabio Berton <fabio.berton@ossystems.com.br>
Date: Fri, 7 Jun 2019 17:25:35 -0300
Subject: [PATCH 1/2] setup.py: remove the setup_requires for setuptools-scm
Organization: O.S. Systems Software LTDA.

The setup_requires argument forces the download of the egg file for setuptools-scm
during the do_compile phase.  This download is incompatible with the typical fetch
and mirror structure.  The only usage of scm is the generation of the _version.py
file and in the release tarball it is already correctly created

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Derek Straka <derek@asterius.io>
Signed-off-by: Fabio Berton <fabio.berton@ossystems.com.br>

%% original patch: 0001-setup.py-remove-the-setup_requires-for-setuptools-sc.patch
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 18d3220..e78ef93 100644
--- a/setup.py
+++ b/setup.py
@@ -23,7 +23,7 @@ INSTALL_REQUIRES = [
 def main():
     setup(
         use_scm_version={"write_to": "src/_pytest/_version.py"},
-        setup_requires=["setuptools-scm", "setuptools>=40.0"],
+        setup_requires=["setuptools>=40.0"],
         package_dir={"": "src"},
         # fmt: off
         extras_require={
-- 
2.20.1

